src/features/expertConsole/rpc-services/api.ts

```diff
@@ -24,30 +23,41 @@ const getMessage = (statusCode: number, isECONNREFUSED: boolean) => {
   }
 }
 
-const apiMethods: any = {}
+const apiRequests: CancelTokenSource[] = []
 
-const api: APIMethods = METHODS.reduce(
-  (obj, method) => ({
-    ...obj,
+type possibleMethods = typeof METHODS[number]
+type dynamicAPI = {
+  [x in possibleMethods]: <T>(params: string[]) => Promise<T>
+}
+
+let apiMethods = {} as dynamicAPI
+
+METHODS.forEach((method: string) => {
+  apiMethods = {
+    ...apiMethods,
+    // eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
```

Comment:
Please do not disable eslint rules. If it complains, please fix/refactor the code to remove the complaint